apps/api/fly.toml

```
@@ -0,0 +1,38 @@
+app = "sunodo-api"
+primary_region = "ord"
+
+# [build]
+# builder = "heroku/buildpacks:20"
+# builder = "paketobuildpacks/builder:base"
+# buildpacks = ["gcr.io/paketo-buildpacks/nodejs"]
+
+[[services]]
+internal_port = 8080
+protocol = "tcp"
+
+[[services.ports]]
+handlers = ["http"]
+port = 80
+force_https = true
+
+[[services.ports]]
+handlers = ["tls", "http"]
+port = 443
```

See https://fly.io/docs/reference/configuration/#the-http_service-section


```suggestion
[http_service]
  internal_port = 8080
  force_https = true
```

Consider adding some concurrency configuration.

```suggestion
[http_service]
  internal_port = 8080
  force_https = true
  [http_service.concurrency]
    type = "requests"
    soft_limit = 200
    hard_limit = 250
```